### DISCUSSION TITLE
How do I enable mouse scroll, but disable click events, like selection?

### DISCUSSION THREAD

**Discussion by JohnHammerSky** [tmux/tmux#4205] [General]
I want to be able to easy copy a text when I select it by double clicking it, which Alacritty provides it, tmux too, but it's annoying because it has some kind of delay and it auto-deselect the text just after the double click event, I need it to behave like Tilix/Alacritty.

I want this behavior on my terminal (Alacritty) + tmux:
- Be able to scroll using mouse wheel
- Use my default terminal behavior for text selection on click

The problem is: 
- If I disable mouse on tmux config, I won't be able to scroll using mouse wheel, it just scrolls my command history
- If I enable mouse on tmux config, it'll let me scroll using mouse wheel, but it'll intercept mouse events like click, not letting my terminal handle the text selection with mouse clicks.

**Comment by nicm** [comment]
https://github.com/tmux/tmux/wiki/FAQ#i-want-to-use-the-mouse-to-select-panes-but-the-terminal-to-copy-how

**Reply by nicm** [reply]
You can adjust tmux's double click behaviour by changing the key binding.

**Reply by JohnHammerSky** [reply]
I can't find the key binding for double click.

**Comment by aless3** [comment]
I made this @JoaoHamerski 

https://github.com/aless3/tmux-click-copy
needs bash (tmux plugin manager needs it anyway)

basically a small delay that cancel the selection if it has not changed(copying it), and copies after you release the mouse button, basically mimics the selection of alacritty with copy-on-select but in tmux so it remains aware of pane boundaries etc.

this keeps working with word selection and line selection (double and triple clicks) 

to completely make it use the same alacritty word separators use also ```set -gw word-separators ",│`|:\"' ()[]{}<>" # same separator as in alacritty```

not exactly what you are looking for, but maybe you wanted this functionality instead of directly using the terminal selection
